modules/virt-example-snippet-nmstate-multiple-interfaces.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippet-nmstate-multiple-interfaces_{context}"]
+= Example snippet: Multiple interfaces in the same Policy
```

Comment:
Maybe remove the word "snippet" from the title, to be consistent with the other examples?

Reply:
Updated titles, ids, and filenames to drop snippet for both offending modules.